api/peering_test.go

```diff
@@ -153,12 +153,13 @@ func TestAPI_Peering_GenerateToken_Read_Initiate_Delete(t *testing.T) {
 	var token1 string
 	// Generate a token happy path
 	resp, wm, err := peerings.GenerateToken(ctx, p1, options)
-	token1 = resp.PeeringToken
```

Comment:
Technically, `resp` can be `nil` here when the endpoint doesn't return in the happy path (aka a test failure)